docs/release-notes/cloud/cloud-2023-09.md

```diff
@@ -0,0 +1,96 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud September 2023.
+included_jira_versions: ['2023.09']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/7.8.2
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/7.10.32
+---
+
+# Cloud September 2023
+
+These release notes are for the Codacy Cloud updates during September 2023.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+<!--TODO Check these issues manually
+
+Jira issues with disabled release notes
+
+Epics:
+-   https://codacy.atlassian.net/browse/ALA-573
+-->
+
+## Product enhancements
+
+-   It's now possible to send notifications to a Slack channel of your choice whenever Codacy detects new critical Security issues in the default branch of any repository in your organization. See [how to install the Codacy Slack integration for Security issues](../../organizations/integrations/slack-integration.md). (TAROT-2242)
+
+    ![Slack integration for Security issues](../images/tarot-2242.png)
+
+-   You can now use the new [Codacy Visual Studio Code extension](https://github.com/codacy/codacy-vscode-extension) to get the full list of problems found by Codacy for a pull request and navigate to any Quality issue that you want to review and fix. See [how to install and use the Codacy Visual Studio Code extension](../../getting-started/integrating-codacy-with-visual-studio-code.md). (ALA-549)
+
+    ![Visual Studio Code extension](../images/ala-549.png)
+
+-   Now, when you switch to a different Enterprise plan or cancel your plan, Codacy no longer re-triggers your repositories analysis automatically. To ensure all information is updated on Codacy UI after changing or canceling your plan, re-analyze your repositories or push new commits. (PLUTO-667)
```

Comment:
I'd also remove this change (PLUTO-667):
1. the change was done to flows available in the Admin panel that aren't available or relevant for customers
2. the transition from a trial or OSS plan, when repositories are not being analyzed, to paid retains its original behavior

Reply:
There has [been changes on retrigerring the analysis](https://www.figma.com/file/lUqbtfF6qvU4y09lc2MlsT/Revert-Active-Pro-Jam?type=whiteboard&node-id=21-408&t=UAaIUzxZRhRgRiRq-4), regardless of Active Pro. 
If we want to include in the release notes, perhaps just emntion the change on the scenarios rather than connecting it to active pro.

Reply:
> is this the user-facing name for Active Pro?
> and is there any reference for the plans where we mention the Enterprise plan?

@mrfyda Yes, we are using "some Enterprise plans" [on the documentation](https://docs.codacy.com/organizations/managing-people/) to refer to the Active Pro plan, as we don't want to be tied to a specific plan name. This was agreed with Amha when we started to document Active Pro specific behavior. We can always review and rethink this terminology, though.

> If we want to include in the release notes, perhaps just emntion the change on the scenarios rather than connecting it to active pro.
@RaquelHipolito I removed the reference to "Enterprise" 👍 

> I'd also remove this change ([PLUTO-667](https://codacy.atlassian.net/browse/PLUTO-667)) 
I included this change because [we decided to document it](https://github.com/codacy/docs/pull/1840).
I'm ok with removing it from the release notes if we don't want to give it too much visibility.

@RaquelHipolito @mrfyda Do you prefer removing PLUTO-667 from the release notes?